Assets/Scripts/RacerAI.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RacerAI : MonoBehaviour, IRacer {
+
+    // Needs to be set at the start of the race to the first waypoint, probably.
+    // i.e., drag first waypoint here in the inspector
```

Comment:
please add a period, I don't understand what this says

Reply:
Frick.

Reply:
alright looks good